src/xml.js

```diff
@@ -49,8 +49,7 @@ define([
                 .replace(/^<xml\b[^>]*>/, "")   // remove <xml ...>
                 .replace(/<\/xml>$/, "")        // remove </xml>
                 .replace(emptytag, "$1 />")     // <tag></tag> to <tag />
-                .replace(/"\/>/g, '" />')       // space before />
-                .replace(/&nbsp;|\xa0/g, " ");  // &nbsp; is not a valid XML entity
+                .replace(/"\/>/g, '" />');      // space before />
```

Comment:
I think this removed the wrong line. Should have removed line 99
https://github.com/dimagi/Vellum/blob/de78396615c7d32d2e2892f2329e27750c72ae27/src/xml.js#L99

Reply:
thanks! whoops 😳